docs/docs.trychroma.com/pages/reference/_sidenav.js

```diff
@@ -1,25 +1,25 @@
 export const items = [
-    {
-      title: 'Python',
-      links: [
-        { href: '/reference/py-client', children: 'Client' },
-        { href: '/reference/py-collection', children: 'Collection' },
-      ]
-    },
-    {
-        title: 'JS',
-        links: [
-          { href: '/reference/js-client', children: 'Client' },
-          { href: '/reference/js-collection', children: 'Collection' },
-        ]
-      },
-      {
-        title: 'Other',
```

Comment:
This is pretty buried / not discoverable. I think it ought to sit also in the migrations page, as well as in the FAQ under something like "my disk is full"

Reply:
I don't think we have a FAQ, but can add it to the migrations page.

Reply:
We have https://docs.trychroma.com/troubleshooting

Reply:
ah cool